oviewer/filter.go

```diff
@@ -40,7 +40,7 @@ func (root *Root) filterDocument(ctx context.Context, searcher Searcher) {
 	r, w := io.Pipe()
 	render, err := renderDoc(m, r)
 	if err != nil {
-		log.Println(err)
+		log.Printf("failed to filter document: %v", err)
```

Comment:
🤔 Just to make sure,  you removed line feed on purpose?

Otherwise, it should be this

```suggestion
		log.Printf("failed to filter document: %v\n", err)
```

I'm raising the point as I already reported other feedbacks on the PR.

Reply:
Thank you. I fixed it because the line breaks in log.Printf were often missing.